### DISCUSSION TITLE
Web Accessibility Annotation Kit Roadmap

### DISCUSSION THREAD

**Discussion by janmaarten-a11y** [cvs-health/annotations#2] [Announcements]
Here are a number of things on our wishlist to explore and add to the Web Accessibility Annotation Kit. They are in no particular order, and unless otherwise noted, there are not yet any specific plans or release dates for these updates.

- Add support for `<figure>` and `<figcaption>` to Image stamps
- Add annotation stamps for page `<title>` tags
- Add annotation components for various [states](https://ericwbailey.website/published/all-the-user-facing-states/)
- Add features for conveniently noting keyboard commands

**Comment by yaaax** [comment]
- Add annotation components for [Content Structure](https://www.w3.org/WAI/tutorials/page-structure/content/) :
  - **list**
  - **article**
  - **section**
  - quote
  - paragraph
  - figure
  - table

Thanks for this wonderful annotation kit!

**Reply by janmaarten-a11y** [reply]
Glad you like the kit @yaaax. Thanks for the suggestions. 

- I believe `<section>` is already supported by the Landmark annotations. 
- We're exploring how we might add better support for `<figure>` and `<figcaption>`. 
- As far as `paragraphs`, `quotes`, `articles`, and `lists`, we felt that those were a bit in the weeds for accessibility specs. In our experience, the necessity of annotating those elements is highly situational. They can still be annotated with the Note, adding code semantics to the details stamp. 
- `Table` is a component that was under consideration. Using Note annotations worked well for our purposes in part because tables can be implemented so many different ways, with such varied types of content, and may not even look like tables. It's hard to account every eventuality with them, and most of the accessibility-specific considerations can be summed up with some code semantics using Note.

**Reply by yaaax** [reply]
Thanks for the reply!

Indeed `<section>` is covered by the Landmark annotation 👍
